### ISSUE TITLE
Convert comments to jsdoc

### ISSUE THREAD

**Issue by Lusito** [Lusito/box2d.ts#14]
Blocked by #13 in order to not make it harder than it already is to port upstream code and comments.

Currently, all comments are in a c++ style, which can't be harnessed by vscode and probably other editors.

All comments should be converted to be in jsdoc style. It's not part of this task to add missing jsdoc. Just convert the existing comments.

It might be necessary to adjust the updiff project a bit after this has been done. Not sure if the updiff project is going to be used much after #13 is done. So maybe it's not worth the effort. We'll have to see.
